client/style.sass

```diff
@@ -1,21 +1,66 @@
+body
+	margin: 0
+	
 #root
-	background-color: cyan
-	height: 100px
+	background-color: #d4e6e3
```

Comment:
Something I'll recommend now before you get too far into styling, is to look into sass variables/partials/imports, and create a root stylesheet with some of these globals that you can import. Then when you want to tweak some color or standard sizing, you only need to do it in one place

Reply:
Created _library.sass which has the colour scheme, main border, etc. Probably spacing in the future, I left a section for it, but didn't include any variables in it for now.